quill-core/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -104,4 +106,11 @@ case class Assignment(input: Ident, property: String, value: Ast)
 
 //************************************************************
 
+sealed trait SchemaDefinition extends Ast
```

Comment:
I'm wondering if we should create new AST types. Do you think we could just change `Entity` and make `Parsing` create an instance of it with the result of the configuration defined using mini-DSL?